src/main/java/io/nats/client/support/JsonValue.java

```diff
@@ -49,6 +49,8 @@ public enum Type {
     public final Object object;
     public final Number number;
 
+    public final List<String> mapOrder;
```

Comment:
helps toString to run through the map keys in a specific order, instead of whatever the keySet iterator would give.